src/instructlab/data/generate.py

```diff
@@ -10,57 +10,55 @@
 from instructlab import configuration as config
 from instructlab import utils
 
-# Local
-from ..utils import http_client
-
 logger = logging.getLogger(__name__)
 
 
 @click.command()
 @click.option(
     "--model",
-    default=config.DEFAULT_MODEL,
-    show_default=True,
+    default=lambda: config.DEFAULTS.DEFAULT_MODEL,
```

Comment:
Most of the `default` in this file should be **not** necessary. They should all get their values from `ctx.default_map`.

Reply:
I've filed an issue for this [here](https://github.com/instructlab/instructlab/issues/1628), let's tackle it in a separate PR.